lib/Models/CswCatalogGroup.js

```diff
@@ -252,7 +261,13 @@ var resourceFormats = [
 ];
 
 CswCatalogGroup.prototype._load = function() {
-    var postDataTemplate = defaultValue(this.getRecordsTemplate, CswCatalogGroup.defaultGetRecordsTemplate);
+    var postDataTemplate;
+
+    if (this.includeWps) {
```

Comment:
The way this is set up, if `includeWps` is true, any WMS items are not included, regardless of the state of `includeWms`.